code/_onclick/hud/new_player.dm

```diff
@@ -256,13 +267,24 @@
 	set_button_status(TRUE)
 	UnregisterSignal(SSticker, COMSIG_TICKER_ENTER_PREGAME)
 
-/atom/movable/screen/lobby/button/settings
+//Subtype the bottom buttons away so the collapse/expand shutter goes behind them
+/atom/movable/screen/lobby/button/bottom
+	layer = LOBBY_BOTTOM_BUTTON_LAYER
 	icon = 'icons/hud/lobby/bottom_buttons.dmi'
+	always_shown = TRUE
+	var/matrix/button_matrix = matrix()
```

Comment:
what? why does this exist

Reply:
let's just say i did what i could vaguely understand and i stopped when it worked like i wanted it to work

Reply:
this is me asking for guidance here as i didn't exactly understand how to use the transform var with animates properly

Reply:
https://github.com/tgstation/tgstation/pull/76443/commits/28ffee160e7e3ecd6602535f6400597692c0b938